clients/rospy/src/rospy/impl/tcpros_service.py

```diff
@@ -154,7 +154,7 @@ def contact_service(resolved_name, timeout=10.0):
             except: # service not actually up
                 if first:
                     first = False
-                    rospy.core.logerr("wait_for_service(%s): failed to contact [%s], will keep trying"%(resolved_name, uri))
+                    rospy.core.logwarn("wait_for_service(%s): failed to contact [%s], will keep trying"%(resolved_name, uri))
```

Comment:
Anyways, I two ways for this (and also the [other comment above](https://github.com/ros/ros_comm/pull/1026#discussion_r112511092)):

1. Publish a warning once when we failed to contact service and then publish a info when it becomes available
2. Publish a throttled info continuously  (and maybe publish a info when it becomes available?)

1\. is easy to miss, but 2. might be a bit spamy. I personally prefer spamy over missing stuff, so I would go for two (with the "now available" info)